docs/source/configuration/key_value_service_configs/postgres_key_value_service_config.rst

```diff
@@ -60,8 +60,64 @@ A minimal AtlasDB configuration for running against postgres will look like :
 
 For more details on the leader block, see :ref:`Leader Configuration <leader-config>`.
 
-Connection parameters
----------------------
+Hikari Connection parameters
+----------------------------
+
+We also provide the following options, which are mapped to `Hikari connection options <https://github.com/brettwooldridge/HikariCP#configuration-knobs-baby>`__:
+
+.. list-table::
+    :widths: 20 20 20 80
+    :header-rows: 1
+
+    *    - Option
+         - Default
+         - Hikari config equivalent
+         - Description
+
+    *    - ``minConnections``
+         - 8
+         - ``minimumIdle``
+         - The number of connections in an idle Hikari pool.
+
+    *    - ``maxConnections``
+         - 256
+         - ``maximumPoolSize``
+         - The maximum size the pool is allowed to reach.
+
+    *    - ``maxConnectionAge``
+         - 1000
+         - ``maxLifetime``
+         - The maximum lifetime, in seconds, of a connection in the pool. ``maxLifetime`` is in ``ms``, so we multiply the provided value by 1000.
+
+    *    - ``maxIdleTime``
+         - 600
+         - ``idleTimeout``
+         - The maximum time, in seconds, a connection may sit idle in the pool. ``idleTimeout`` is in ``ms``, so we multiply the provided value by 1000.
```

Comment:
It's a bit unfortunate that this is in seconds whilst `checkoutTimeout` in our own config is in milliseconds, but I guess changing now would break users.